src/main/java/school/lemon/changerequest/java/introduction/pr1/Bob.java

```diff
@@ -4,7 +4,28 @@
 public class Bob {
 
     public static String hey(String phrase) {
-        return "";
+        char ch;
+        int counter = 0;
+        boolean flag = true;
+        String reply = "";
+        if(phrase == "") return "Fine.";
+
+        for(int i = 0; i < phrase.length(); i++){
+            ch = phrase.charAt(i);
+            if(ch == '?'){
+                reply = "Yeap.";
+            } else if(ch == '!'){
```

Comment:
It's not correct case for yelling. Check the test cases and determine what should be in the phrase for each type of answer.